format/FormatUtils.js

```diff
@@ -4,27 +4,19 @@
  *
  * Copyright © 2021 Extremely Heavy Industries Inc.
  */
-import {defaults, isPlainObject} from 'lodash';
 
 /**
  * Generate a renderer for a given formatter function.
  *
  * The returned renderer takes an optional config for its assigned formatter and itself returns
  * a function that takes only a value, for convenient use with e.g. grid cell rendering.
  *
- * If the formatter config is an object it will be cloned before being passed to its formatter
- * to ensure that the formatter gets its own clean copy each time it is called.
- *
  * @param {function} formatter - an existing formatter method.
  * @return {function(v: Object):function} - a configurable renderer.
  */
 export function createRenderer(formatter) {
     return function(config) {
-        const isObj = isPlainObject(config);
-        return v => {
-            const formatterConfig = isObj ? defaults({}, config) : config;
-            return formatter(v, formatterConfig);
-        };
```

Comment:
Seems _possible_ that some app-specific renderer was counting on this behavior, but not enough for me to think we should really call it out or worry about it.  Noting it here only so we can keep it in mind / keep an eye out for any trouble reports.